src/url-analytics/url-analytics.service.ts

```diff
@@ -47,4 +132,27 @@ export class UrlAnalyticsService {
 		const location = lookup(ip);
 		return location?.country;
 	}
+
+	private async parseQuerytoArray(queryKey: string): Promise<string[]> {
+		const queryKeyArray: string[] = queryKey.includes(',') ? queryKey.split(',') : [queryKey];
```

Comment:
```suggestion
		const queryKeyArray: string[] = queryKey.split(',');
```

This should work fine as well

Reply:
Done dai